server/src/main/java/com/google/tsunami/security/scanner/utilities/TsunamiTestbedServer.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.tsunami.security.scanner.utilities;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.ParameterException;
+import io.grpc.Server;
+import io.grpc.ServerBuilder;
+import io.kubernetes.client.openapi.ApiClient;
+import io.kubernetes.client.openapi.Configuration;
+import io.kubernetes.client.util.Config;
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+/** Builds and starts a GRPC-based Tsunami Testbed server. */
+public final class TsunamiTestbedServer {
+
+  private static final int DEFAULT_PORT = 8000;
+
+  private static class TsunamiTestbedServerArgs {
+    @Parameter(
+        names = "--port",
+        description = "The port on which the server listens.",
+        validateWith = ValidPort.class)
+    private int port = DEFAULT_PORT;
+
+    @Parameter(
+        names = {"--help", "-h"},
+        description = "Print parameters and description.",
+        help = true)
+    private boolean help = false;
+
+    public int getPort() {
+      return port;
+    }
+
+    public boolean isHelp() {
+      return help;
+    }
```

Comment:
Same here, make members public and annotate class with `@Parameters` annotation.